### PR TITLE
content: Add third-party JS intro post

### LINKED ISSUE #1096: content: Third-party JavaScript performance
**A one to two sentence description of your post**
Introduction for third-party JavaScript optimization guides. Explains what is third-party JS and how it affects performance.

**Target publish date:** `<2019-07-18>`
- [ ] Check this box if this is a hard deadline.

**Process**
- [x] 1. I've done my proposal https://docs.google.com/document/d/1KiBjSqCnR5DeDs1H66uE5hna0baKOwbbrg72_TScuK8/edit?usp=sharing
- [x] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc
- [x] 4. Draft approved — _reviewer checks this_
- [x] 5. I've submitted a pull request #1142
- [x] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by mfriesenhahn**
Fixes #1096 

Changes proposed in this pull request:

- Adds "Third-party JavaScript performance" post.

### CODE REVIEW

**Comment by mihajlija** [comment]
I consent.

File: src/site/content/en/fast/third-party-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+{% endAside %}
+
+## What to do about it
+Using third-party JavaScript is often unavoidable, but there are things you can do to minimize adverse effects:
+- When choosing third-party resources, favor those that send the least amount of code while still giving you the functionality you need.
+- Use [performance budgets](https://web.dev/use-lighthouse-for-performance-budgets/) for third-party content to keep their cost in check.
+- Don't use the same functionality from two different vendors. You probably don't need two tag managers or two analytics platforms.
+- Routinely audit and clean out redundant third-party scripts.
+
+To learn how to audit third-party content and load it efficiently for better performance and user experience, check out the other posts in the [third-party resources collection](/fast/#optimize-your-third-party-resources).
```
Comment (mfriesenhahn) [inline src/site/content/en/fast/third-party-javascript/index.md]:
```suggestion
To learn how to audit third-party content and load it efficiently for better performance and user experience, check out the other posts in the [Optimize your third-party resources ](/fast/#optimize-your-third-party-resources) section.
```
Nit-picky edit: we're trying to keep site nomenclature consistent. So, we're reserving "collection" for the top-level groupings (fast, reliable, etc.).

**Comment by mihajlija** [comment]
Planning to use it (as per our email) but I uploaded the wrong file, the correct one is on my home PC. This one has poor timing, I noticed when I got to the office and ran a preview build, but can't fix it until I get home. 

Re: hero image: if I crop the margins, the thumbnail will look good, but in the "hero view" the content gets cut-off on higher resolutions. This sucks because my hero is "contentful" and serves as a teaser for the animated image. Not having a thumbnail would be even worse imo. 

Is there a way to use different images for these two purposes? Or a way to keep only the thumbnail?

**Comment by robdodson** [comment]
> Is there a way to use different images for these two purposes? Or a way to keep only the thumbnail?

Not at the moment but I think it's worth us adding both of these behaviors. I'm in all day trainings until Thursday this week but I could work on adding these behaviors later in the week if you're ok waiting?

**Comment by mihajlija** [comment]
Sounds great, thanks! I think having only the thumbnail and no hero image will be a good choice for many posts which start in /blog and transition to a learning path.

Btw, I was under the impression that nothing gets merged until the end of the sprint? Either way, I'll take care of the video today 👍

**Comment by robdodson** [comment]
> Btw, I was under the impression that nothing gets merged until the end of the sprint? Either way, I'll take care of the video today 👍

Sorry for any confusion. We merge as soon as a post is ready and ideally deploy it soon after it's been merged. For the third-party posts we were thinking of merging them and launching them all at the same time since they're related.

**Comment by mfriesenhahn** [comment]
> * Is the third-party-examples.mp4 file used?
> * The hero image looks a bit odd on the blog landing page when compared against other posts
>   image
> 
> I'd suggest swapping it to one that's full size or removing the hero entirely. @mfriesenhahn do you have an opinion on which you'd prefer?

I'd prefer replacing. And thanks for the catch!

**Comment by robdodson** [comment]
I had to defer the picture element work to the next sprint. I think I would suggest not using a hero image for this post (there are a few other ones on the blog that don't use a hero) and we can revisit this next sprint if we want to add a hero then. #1199

**Comment by robdodson** [comment]
@mihajlija I've assigned this back to you to remove the hero image. You can also add a `draft: true` field to the YAML so we can merge the post without it going live.

**Comment by mihajlija** [comment]
@robdodson done ✓

**Comment by robdodson** [comment]
Rad thank you!

**Pull request merged**